solutions/one-liners/one-liners.py

```diff
@@ -0,0 +1,25 @@
+"""
+Can you write a one-liner for the following code snippet?
+
+def puns(a, b, c):
+    if a==b or b<c:
+        return "Did you hear about the guy whose whole left side was cut off? He's all right now."
+    elif a==b and a==c:
+        ######### don't think this case will ever happen
+        return "I'm reading a book about anti-gravity. It's impossible to put down."
+    elif not b!=c:
+        ######### logically, if b == c
+        return "Yesterday I accidentally swallowed some food coloring. The doctor says I'm OK, but I feel like I've dyed a little inside."
+    else:
+        return "The roundest knight at king Arthur's round table was Sir Cumference."
+
+print puns(1, 2, 3)
+"""
+
+
+print (lambda a, b, c:
```

Comment:
Great work!

However this lambda is the one-liner for the if/else construct. This however doesnt convert `puns` itself into a one-line function like the other examples do.

Essentially after the solution one-liner is run in a python shell, you should be able to then execute fucntion `puns` on different argument values. In command line shell, you should be able to do something like:
```python
>>> (lambda __print, _ ... ...  ...
>>> puns(3, 4, 100)
"Did you hear about the guy whose whole left side was cut off? He's all right now."
>>> puns(3, 4, 100)
"The roundest knight at king Arthur's round table was Sir Cumference."
```
You can try the examples out and you'll see what I mean.